datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/handler/DatalakeFullBackupWaitHandler.java

```diff
@@ -64,10 +64,14 @@ protected Selectable doAccept(HandlerEvent<DatalakeFullBackupWaitRequest> event)
         Long sdxId = request.getResourceId();
         String userId = request.getUserId();
         Selectable response;
-        int duration = entitlementService.isLongTimeBackupEnabled(ThreadBasedUserCrnProvider.getAccountId())
-                ? longDurationInMinutes : durationInMinutes;
+        int duration;
+        if (entitlementService.isLongTimeBackupEnabled(ThreadBasedUserCrnProvider.getAccountId())) {
```

Comment:
Now this is quite bad pattern. You provide an API parameter which in case a customer have an entitlement does not have any meaning. If you want to support the entitlement and the prooper way to handle this is to give back a BadRequest to notify that with this entitlement your value is not accepted. Or at least log a warn message that this settings was overridden. 
Or even it might be cheaper to say that if a value is provided, than the entitlement is meaningless.

Reply:
Hi @lnardai , thanks a lot for this great suggestion! I talked with Kalyan and we decided to change behavior of this on isLongTimeBackupEnabled: if a configured value is passed in, the passed in value will override the value of isLongTimeBackupEnabled. Please let me know if the updated code looks good. Thank you!!

Reply:
@lnardai could you please check the changes?